cmd/skuba/skuba.go

```diff
@@ -49,14 +51,29 @@ func newRootCmd() *cobra.Command {
 }
 
 func main() {
-	fmt.Println("** This is a BETA release and NOT intended for production usage. **")
+	printVersionStatement()
 	klog.InitFlags(nil)
 	cmd := newRootCmd()
 	if err := cmd.Execute(); err != nil {
 		os.Exit(1)
 	}
 }
 
+func printVersionStatement() {
+	switch {
+	case skubapkg.Tag == "":
+		fmt.Println("** This is an UNTAGGED version and NOT intended for production usage. **")
+	case strings.Contains(skubapkg.Tag, "alpha"):
```

Comment:
right now we have tagged releases without suffixes that will report as "production" releases. 
Maybe we can also check for versions starting with 0 (ie 0.9.4) and report those as alpha as a catch all, then release GMC as v1.0.0.

Reply:
I didn't want to care about the past because this PR will be effective once it merges; at that point is when we can start tagging with the new method. Am I misunderstanding?

Reply:
In fact I did misunderstand, sorry. We are about to release a couple of RC's and after that we'll go with a "final" (no suffix) version -- that doesn't need to be `1.0.0`.

All things considered I don't think the benefit for this intermediate time is too much to explicitly catch the `Major() == 0` case (besides, we have released `0.X.Y` as alphas, betas and rc's at this time, so `Major() == 0` is not giving a "maturity" level).